examples/gw_incompressible_3d.py

```diff
@@ -0,0 +1,162 @@
+from gusto import *
```

Comment:
Do we need another example file? What does this test that sk_nonlinear_3d doesn't?